component/app.js

```diff
@@ -0,0 +1,40 @@
+class App {
+    constructor(gradeTable) {
+        this.gradeTable = gradeTable;
+        
+        this.handleGetGradesError = this.handleGetGradesError.bind(this);
+        this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+
+    }
+
+
+    handleGetGradesError(error) {
+        console.error(error);
+    }
+
+    handleGetGradesSuccess(grades) {
+        console.log("grades: ", grades);
+
+        this.gradeTable.updateGrades(grades);
+    }
+
+    getGrades () {
+    
+        var ajaxOptions = {
+        url: "https://sgt.lfzprototypes.com/api/grades",
+        method: "GET",
+        dataType: "json",        
+        headers: {
+            "X-Access-Token": "944Roegi"
+        },
+        success: this.handleGetGradesSuccess,
+        error: this.handleGetGradesError
+    };
+
+        return $.ajax(ajaxOptions);
+    }

```

Comment:
Also, I'd try to ensure your indenting is consistent. It'll make reviewing the structure of your code much easier later on (either by you or by your co-workers).